### PR TITLE
support ariadne recommendations

### PR DESCRIPTION
**Pull request by linpengzhang**
adapt the code to the new LwJson format which now supports Ariadne recommendations

Tested on https://github.com/lacework-dev/code-analysis-test-repo/pull/307

### CODE REVIEW

File: src/lw-json.ts [hunk 1]
```diff
@@ -2,18 +2,41 @@ export interface LWJSON {
   FixSuggestions?: FixSuggestion[]
 }
 
-// Fix suggestion
+export interface VersionEdge {
```
Comment (PatGod) [inline src/lw-json.ts:5]: Could we please rename 'VersionEdge' by 'NextNode' as in lwjson.go?

File: src/lw-json.ts [hunk 2]
```diff
@@ -2,18 +2,41 @@ export interface LWJSON {
...
+  Version: string
+  Severity?: string
+  CVE?: string
+}
 
-export interface FixVersion {
-  Type?: string
-  Version?: string
+export interface VersionNode {
+  Version: string
```
Comment (PatGod) [inline src/lw-json.ts:12]: Please rename 'VersionNode' by 'SortedVersionGraphNode' as in lwjson.go

File: src/lw-json.ts [hunk 3]
```diff
@@ -2,18 +2,41 @@ export interface LWJSON {
...
+  Edges?: VersionEdge[]
+}
+
+export interface VersionGraphInfo {
+  NodeCnt?: number
```
Comment (PatGod) [inline src/lw-json.ts:17]: Please rename ' VersionGraphInfo' by 'VersionGraphStruct' as in lwjson.go

File: src/lw-json.ts [hunk 4]
```diff
@@ -2,18 +2,41 @@ export interface LWJSON {
...
+  EdgeCnt?: number
+  ExtraAthenaCalls?: number
+  VersionGraph?: VersionNode[]
+}
+
+export interface Recommendation {
+  FixVersion: FixVersion
```
Comment (PatGod) [inline src/lw-json.ts:24]: Please replace 'Recommendation' by 'FixInfo' as in lwjson.go?

File: src/lw-json.ts [hunk 5]
```diff
@@ -2,18 +2,41 @@ export interface LWJSON {
...
+  Kind?: string
 }
 
 export interface Info {
-  FixVersion?: FixVersion
```
Comment (PatGod) [inline src/lw-json.ts:13]: Please rename 'Info' by 'FixSuggestionInfo' as in lwjson.go

**Review by PatGod — CHANGES_REQUESTED** [review]
Could you please perhaps address these minor cosmetic suggestions?
It would be nice to keep the same terms as used in lwjson.go, as much as possible.
Thanks!

**Comment by linpengzhang** [comment]
> Could you please perhaps address these minor cosmetic suggestions? It would be nice to keep the same terms as used in lwjson.go, as much as possible. Thanks!

Thanks for the carefully review, I have addressed the comments and ensured the test passes again!

**Review by PatGod — APPROVED** [review]

**Pull request merged**